scripts/sd_merge_csv.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
We need to choose a specific version of python (2.7 or 3.6 being the most likely choices) and hard code it here. `python` is too ambiguous these days :/

Reply:
OK, I've gone with 2.7. In the V8 source I see `3` and `2.4`, FWIW.

Reply:
2.4 -- hohoho, I really hope that's some script that they never actually run :) For a while I thought `python3` was OK, but as the Python 3 versions are changing more fundamentally than I expected I've ended up with `python3.x`. This will be annoying in the future, but oh well.

Anyway, I think `python2.7` is the right choice here.